server/utils/nunjucksSetup.ts

```diff
@@ -11,7 +11,7 @@ export default function nunjucksSetup(app: express.Express): void {
   app.set('view engine', 'njk')
 
   app.locals.asset_path = '/assets/'
-  app.locals.applicationName = 'Hmpps Contacts Ui'
+  app.locals.applicationName = 'Digital Prison Services'
```

Comment:
not sure if we should change the application name here?

Reply:
Hmpps Contacts Ui added back